packages/deployment/upgrade-test/Dockerfile

```diff
@@ -57,7 +57,7 @@ RUN . ./upgrade-test-scripts/start_to_to.sh
 
 ARG DEST_IMAGE
 #this is agoric-upgrade-10 / vaults
-FROM ghcr.io/agoric/agoric-sdk:34 as agoric-upgrade-10
+FROM ghcr.io/agoric/agoric-sdk:35 as agoric-upgrade-10
```

Comment:
confirmed: `agoric-sdk:35` is built from `mainnet1B-rc3`

 - https://github.com/Agoric/agoric-sdk/releases/tag/mainnet1B-rc3 is ba3b776
 - https://github.com/Agoric/agoric-sdk/releases/tag/v0.34.1 is also ba3b776

 - https://github.com/agoric/agoric-sdk/pkgs/container/agoric-sdk/100269055?tag=20230609063312-ba3b77
has `sha256:9e4743ec7c60f1bb6ae9de7decd22ca4df8983f9195e19ffc0541a8645e1e71b`
as does
 - https://github.com/agoric/agoric-sdk/pkgs/container/agoric-sdk/100269055?tag=35